guacamole-docker/bin/start.sh

```diff
@@ -659,9 +659,15 @@ associate_header() {
 ##
 start_guacamole() {
 
+    # User-only writable CATALINA_BASE
+    export CATALINA_BASE=$GUACAMOLE_HOME/tomcat
+    for dir in logs temp webapps work; do
+        mkdir -p $CATALINA_BASE/$dir
+    done
+    cp -R /usr/local/tomcat/conf $CATALINA_BASE
```

Comment:
Will a symbolic link not work in this case?

Reply:
Well, it would work.
But at least one other PR plans to modify the configuration : #489.
Let's then keep it writable 👍